test/integration/answer_test.rb

```diff
@@ -5,6 +5,7 @@ class AnswerTest < ActionDispatch::IntegrationTest
     setup_and_visit_content_item('answer')
     assert page.has_text?(@content_item["title"])
     assert_has_component_govspeak(@content_item["details"]["body"])
+    assert page.has_text?("Last Updated: 28 Mai 2015")
```

Comment:
Should `Last Updated:` be welsh too?
https://github.com/alphagov/frontend/blob/master/config/locales/cy.yml#L3